packages/render-fidelity-tools/test/config.json

```diff
@@ -709,6 +869,22 @@
         "dspbr-pt"
       ]
     },
+    {
+      "name": "khronos-SimpleSkin",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/SimpleSkin/glTF/SimpleSkin.gltf",
+      "orbit": {
+        "radius": 2.5
+      },
+      "target": {
+        "y": 1
+      },
+      "dimensions": {
+        "height": 1400
```

Comment:
since this test isn't very interesting, let's reduce the width rather than increase the height.